web/src/components/overlay/detail/SearchDetailDialog.tsx

```diff
@@ -631,13 +709,30 @@ function ObjectDetailsTab({
           {search?.data.recognized_license_plate && (
             <div className="flex flex-col gap-1.5">
               <div className="text-sm text-primary/40">
-                Recognized License Plate
+                {t("details.recognizedLicensePlate")}
               </div>
               <div className="flex flex-col space-y-0.5 text-sm">
                 <div className="flex flex-row items-center gap-2">
                   {search.data.recognized_license_plate}{" "}
                   {recognizedLicensePlateScore &&
                     ` (${recognizedLicensePlateScore}%)`}
+                  <Tooltip>
```

Comment:
do we make the pencil icon only available for admin users?

Reply:
Oh yeah, good call. I'll do that for sub labels too